src/collection-view.js

```diff
@@ -1,16 +1,16 @@
 // Collection View
 // ---------------
 
-import _                  from 'underscore';
-import Backbone           from 'backbone';
-import destroyBackboneView from './utils/destroy-backbone-view';
-import isNodeAttached     from './common/is-node-attached';
-import childViewEventHandler from './common/child-view-event-handler';
-import monitorViewEvents  from './common/monitor-view-events';
-import { triggerMethodOn } from './common/trigger-method';
-import ChildViewContainer from './child-view-container';
-import MarionetteError    from './error';
-import ViewMixin          from './mixins/view';
+import _                      from 'underscore';
```

Comment:
soooo.. we haven't enforced this or fixed it or even documented it.. but somewhere along the way of v3 in gitter we chatted about not aligning the imports.  There is definitely some aligning still around, but I think the predominate argument against keeping up the alignment was that adding 1 import may mean a change to every import line and it just jacks with git blame.